libcudacxx/include/cuda/std/__type_traits/is_unsigned_integer.h

```diff
@@ -20,32 +20,31 @@
 #  pragma system_header
 #endif // no system header
 
-#include <cuda/std/__type_traits/integral_constant.h>
-
 _LIBCUDACXX_BEGIN_NAMESPACE_STD
 
-template <class _Tp>
-struct __cccl_is_unsigned_integer : public false_type
-{};
+template <typename>
+inline constexpr bool __cccl_is_unsigned_integer_v = false;
```

Comment:
We still need `_CCCL_INLINE_VAR` because not all compilers properly implement inline variables